css/style.css

```diff
@@ -0,0 +1,189 @@
+
+.body {
+  max-width: 1440px;
+  background: white;
+  margin: 0 60px 24px 42.72px;
+  font-family: "Montserrat";
+  color: #000;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  height: 109px;
+  align-items: center;
+}
+
+.header__img {
+  display: block;
+}
+
+.bloks-1 {
+  flex-grow: .05;
+}
+
+.bloks-2 {
+  flex-grow: .35;
+}
+
+.bloks-3 {
+  flex-grow: .065;
+}
+
+.car__title {
+  text-align: center;
+  font-slyle: normal;
+  font-weight: 600;
+  font-size: 90px;
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+.car__subtitle {
+  text-align: center;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 15px;
+  margin-top: 0;
+}
+
+.car__tsl {
+  display: block;
+  height: 450px;
+  margin: auto;
+}
+
+.car {
+  background: url('../IMG/tesla bgr.png') no-repeat;
+  height: 100%;
+}
+
+.header__menu {
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.header__menu_link {
+  color: #000000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/teslaproject/pullRequest?prid=5566861)